content/packages.md

```diff
@@ -0,0 +1,386 @@
+
+`api.use` is for recording the dependencies of your package. These dependencies are only available inside the package. However, Meteor's package system is single-loading so no two packages in the same app can have dependencies on conflicting versions of a single package. Read more about that in the section about version constraints below.
+
+```js
+// Example of using api.use to register dependencies internal to the package
+api.use([
+  'ecmascript',
+  'check',
+  'ddp',
+  'underscore',
+  'aldeed:simple-schema@1.3.3',
+  'mdg:validation-error@0.1.0'
+]);
+```
+
+`api.imply`, on the other hand, does not include the package as an internal dependency; instead, it makes these packages available to the user of the package, as if they were exported from this one. In the example below, we use `api.imply` to include Flow Router in the `todos-lib` package, so that any user of `todos-lib` also gets to use the symbol `FlowRouter`. This can be helpful to avoid keeping long lists of dependencies up to date by creating meta-packages that encapsulate many dependencies of your app at once.
+
+```js
+// Example of using api.imply to make a meta-package
+// Taken from the todos-lib package in the Todos example app
+api.imply([
+  'kadira:flow-router@2.7.0',
+  'kadira:blaze-layout@2.2.0',
+  'arillo:flow-router-helpers@0.4.5',
+  'zimme:active-route@2.3.0',
+]);
+```
+
+`api.use` and `api.imply` can also take an extra argument to specify the architecture on which these dependencies should be used - this way, you can register a dependency on only the server version of HTTP, for example. This can be helpful when you are trying to reduce the side of your client-side app bundle.
+
+### You can't depend on a specific Meteor version
+
+Note that the Meteor release version number is mostly a marketing artifact, so it doesn't follow SemVer. This means packages can only depend on specific versions of the packages inside a Meteor release, but can't depend on a specific release itself. We have a helpful shorthand api called `versionsFrom` that handles this for you by automatically filling in package version numbers from a particular release:
+
+```js
+// Use versions of core packages from Meteor 1.2.1
+api.versionsFrom('1.2.1');
+
+api.use([
+  'ecmascript', // Don't need to specify version because of versionsFrom above
+  'check',
+  'aldeed:simple-schema@1.3.3',
+  'mdg:validation-error@0.1.0'
+]);
+```
+
+The above code snippet is equivalent to the code below, which specifies all of the version numbers individually:
+
+```js
+api.use([
+  'ecmascript@0.1.6',
+  'check@1.1.0',
+  'aldeed:simple-schema@1.3.3',
+  'mdg:validation-error@0.1.0'
+]);
+```
+
+## Architectures
+
+Meteor packages are built around the idea of multiple architectures where code might run. Here are all currently possible architectures:
+
+
+- `web` or `client` - code that runs in a web browser; can be split between Cordova and browser.
```

Comment:
Maybe like "Cordova (code running in Meteor's mobile container)"? I might not know what Cordova is